### ISSUE TITLE
Educating users on how to secure their accounts

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#2058]
The only advice we currently give users is “Keep in mind that Liberapay is a financial service, use a strong password!”. That's much too short and ambiguous.

I think we need a dedicated website to educate people about password managers. It would be a generic and neutral website that any web application can link to, i.e. it wouldn't be specific to Liberapay and it wouldn't recommend only one password manager.

Similar issue on a different topic: #720.

**Comment by miku86** [comment]
I don't think this is an educational problem, so I also don't think that sending people to other resources would solve this problem, as we can see in the millions of bad passwords out there. 

I think convenience is the biggest hurdle, that's why I'm for better security policies, e.g. by increasing `PASSWORD_MIN_SIZE = 8` to `10` or `12`. And if people have problems creating a good one, we can support them with links to educational sites, but only as a hint.

**Comment by DeclanChidlow** [comment]
I've written something like this before, and would like to take this on, but I'm not quite sure how extensive it should be in this case.

Obviously advising password managers should be included as mentioned, but do you think it would be wise to also include information on scams, and perhaps how to verify if an email really is from Liberapay?

Perhaps also a mention of securing associated payment processors?

**Comment by Changaco** [comment]
The idea was to create a generic and neutral web page that any web application could link to, so there shouldn't be anything specific to Liberapay in it.

I think the minimum that needs to be covered is passwords and multi-factor authentication. Additional topics could be covered, perhaps in separate pages. The explanations should be understandable by pretty much anyone without requiring prior knowledge.
